src/main/java/com/happeningnow/service/ServiceOrganizer.java

```diff
@@ -0,0 +1,41 @@
+package com.happeningnow.service;
+
+import com.happeningnow.model.Organizer;
+import com.happeningnow.repository.OrganizerRepository;
+import jakarta.transaction.Transactional;
+import lombok.AllArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.stereotype.Service;
+import java.util.Optional;
+import java.util.UUID;
+
+@AllArgsConstructor
+
+@Service
+public class ServiceOrganizer {
+
+    @Autowired
+    private final OrganizerRepository organizerRepository;
```

Comment:
please make this class receive this as a parameter, just like I showed you the other day.

Reply:
I don´t remember this very well. I go study to make